src/SDKs/TrafficManager/TrafficManager.Tests/TrafficManager.Tests.csproj

```diff
@@ -4,16 +4,15 @@
     <PackageId>TrafficManager.Tests</PackageId>
     <Description>TrafficManager.Tests Class Library</Description>
     <AssemblyName>TrafficManager.Tests</AssemblyName>
-    <VersionPrefix>1.0.0-preview</VersionPrefix>    
+    <VersionPrefix>3.0.0</VersionPrefix>    
```

Comment:
@vladca this is unnecessary as you never publish your test nuget package.
So please revert this change.